integration_tests/mockApis/sendLegalMail.ts

```diff
@@ -1,625 +0,0 @@
-import { SuperAgentRequest } from 'superagent'
```

Comment:
This one file contained all of the wiremock stubs for our cypress tests for the Send Legal Mail API; and had turned into a bit of a monster containing stubs for link, barcode, contact and cjsm related endpoints. 
Rather than add yet more for One Time Code related endpoints, I have split this file up into the individual files you see further down in this PR.
All of the 'new' files in this PR are just sections of this file moved out into files in their own right